libraries/Microsoft.Bot.Builder.Dialogs/DialogExtensions.cs

```diff
@@ -95,6 +95,17 @@ public static async Task RunAsync(this Dialog dialog, ITurnContext turnContext,
             }
         }
 
+        /// <summary>
+        /// Obtain the CultureInfo in DialogContext.
+        /// </summary>
+        /// <param name="dialogContext">The dialogContext to extract information.</param>
+        /// <returns>A string representing the current locale.</returns>
+        public static string GetLocale(this DialogContext dialogContext)
+        {
```

Comment:
We don't need an extension method for this, this method can be added to DialogContext that is in the same assembly as this extension method. Move it there

Reply:
Moved it to DialogContext class